src/enrichment/base.rs

```diff
@@ -265,3 +268,11 @@ pub async fn run_enrichment_worker<T: EnrichmentWorker>(
 
     Ok(())
 }
+
+/// Schema for what a survey match entry looks like in an enriched alert.
+#[derive(serde::Deserialize, serde::Serialize, Debug, Clone, AvroSchema, JsonSchema)]
+pub struct SurveyMatch {
+    pub object_id: String,
+    pub prv_candidates: Vec<PhotometryMag>,
```

Comment:
Happy to give a go a this, but as we discussed earlier I think we want to swap the super simplified `PhotometryMag` w/ the much more detailed and usable photometry model we use in the output of the filter worker (or something similar at least, where we include zeropoints, whether is a positive or negative detection, ...). But again I'm happy to make this part happen